### PR TITLE
Tests converted to Jest

### PR DESCRIPTION
**Pull request by msk-**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,23 +37,34 @@
...
+        "functions": 18,
+        "branches": 17,
+        "lines": 30
+      }
+    }
+  },
   "devDependencies": {
     "eslint": "6.3.0",
-    "istanbul": "1.1.0-alpha.1",
+    "jest": "^24.9.0",
```
Comment (rmothilal) [inline package.json]: please remove caret i.e. ^

File: package.json [hunk 2]
```diff
@@ -37,23 +37,34 @@
...
   },
+  "jest": {
+    "collectCoverageFrom": [
+      "**/src/handlers/**/*.js",
+      "src/data/database.js",
+      "**/src/model/**/*.js"
+    ],
+    "coverageThreshold": {
+      "global": {
+        "statements": 30,
```
Comment (rmothilal) [inline package.json]: please update to 90 for all coverage percentage

**Comment by rmothilal** [comment]
Also unit tests are failing

File: package.json (hunk 1 above)
Comment (mdebarros) [inline package.json]: Please remove carrot

**Pull request merged**